src/framework/components/rigid-body/system.js

```diff
@@ -46,6 +47,13 @@ class RaycastResult {
          * @type {Vec3}
          */
         this.normal = normal;
+
+        /**
+         * The distance percentage at which the collision occurred from starting point.
```

Comment:
When you say 'percentage', is this a normalized value (0..1) or is it in a different range like 0.100?

Reply:
It is normalized. Between 0 and 1.

Reply:
Cool. Can you update the JSDoc to mention that please?

Reply:
It's updated!